lib/fastlane/plugin/sentry/actions/sentry_upload_dif.rb

```diff
@@ -0,0 +1,163 @@
+        command.push('--require_all') unless params[:require_all].nil?
+        command.push('--symbol_maps').push(params[:symbol_maps]) unless params[:symbol_maps].nil?
+        command.push('--derived_data') unless params[:derived_data].nil?
+        command.push('--no_zips') unless params[:no_zips].nil?
+        command.push('--info_plist').push(params[:info_plist]) unless params[:info_plist].nil?
+        command.push('--no_reprocessing') unless params[:no_reprocessing].nil?
+        command.push('--force_foreground') unless params[:force_foreground].nil?
+        command.push('--include_sources') unless params[:include_sources].nil?
+        command.push('--wait') unless params[:wait].nil?
+        command.push('--upload_symbol_maps') unless params[:upload_symbol_maps].nil?
+
+        Helper::SentryHelper.call_sentry_cli(command)
+        UI.success("Successfully ran upload-dif")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Upload debugging information files."
+      end
+
+      def self.details
+        [
+          "Files can be uploaded using the upload-dif command. This command will scan a given folder recursively for files and upload them to Sentry.",
+          "See https://docs.sentry.io/product/cli/dif/#uploading-files for more information."
+        ].join(" ")
+      end
+
+      def self.available_options
+        Helper::SentryConfig.common_api_config_items + [
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       description: "A path to search recursively for symbol files"),
+          FastlaneCore::ConfigItem.new(key: :type,
+                                       short_option: "-t",
+                                       description: "Only consider debug information files of the given \
+                                       type.  By default, all types are considered",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                        UI.user_error! "Invalid value '#{value}'" unless ['dsym', 'elf', 'breakpad', 'pdb', 'pe', 'sourcebundle', 'bcsymbolmap'].include? value
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :no_unwind,
+                                       description: "Do not scan for stack unwinding information. Specify \
+                                       this flag for builds with disabled FPO, or when \
+                                       stackwalking occurs on the device. This usually \
+                                       excludes executables and dynamic libraries. They might \
+                                       still be uploaded, if they contain additional \
+                                       processable information (see other flags)",
```

Comment:
`l`: I think we are missing the `.` for almost all descriptions.
```suggestion
                                       processable information (see other flags).",
```

Reply:
Last time i checked the fastlane plugin threw an error when adding trailing `.` in descriptions.

Reply:
Yep. 

<img width="994" alt="Bildschirmfoto 2021-07-27 um 11 45 40" src="https://user-images.githubusercontent.com/3984453/127133556-2adec334-750c-402e-989e-dcfe31aaf04a.png">